### DISCUSSION TITLE
Other forms of notification?

### DISCUSSION THREAD

**Discussion by nsmela** [qatrackplus/qatrackplus#626] [Ideas]
Hey,

How hard would it be to implement an audio notification instead of using emails? Could it be similar to the system where websites ask to allow notifications, like Facebook?

**Comment by randlet** [comment]
Not sure!  I've never implemented notifications like that but if you google for Django real time notifications, or Django "browser tab notifications" or something like that I suspect it's a problem others have solved before.  I'm not personally interested in implementing this (I find this kind of notification distracting) but if it's something you (or someone else) were interested in implementing I'd be happy to review a pull request.
